notebooks/04_basic_parameters_tuning.ipynb

```diff
@@ -94,14 +94,15 @@
     "    'workclass', 'education', 'marital-status', 'occupation',\n",
```

Comment:
<p>The same here. Using <code>os.path.join</code>  will be safer.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lesteve/scikit-learn-tutorial/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>